Observer/SendPurchaseEvent.php

```diff
@@ -44,32 +57,31 @@ public function __construct(
      */
     public function execute(Observer $observer)
     {
-        if (!$this->scopeConfig->getValue(GAClient::GOOGLE_ANALYTICS_SERVERSIDE_ENABLED, ScopeInterface::SCOPE_STORE)) {
-            return;
-        }
-        $ua = $this->scopeConfig->getValue(GAClient::GOOGLE_ANALYTICS_SERVERSIDE_UA, ScopeInterface::SCOPE_STORE);
-        if (!$ua) {
-            $this->logger->info('No Google Analytics account number has been found in the ServerSideAnalytics configuration.');
+        /** @var \Magento\Sales\Model\Order\Payment $payment */
+        $payment = $observer->getPayment();
+        /** @var \Magento\Sales\Model\Order $order */
+        $order = $payment->getOrder();
+
+        $this->emulation->startEnvironmentEmulation($order->getStoreId(), 'adminhtml');
```

Comment:
This has been added to support getting the GA config & UA codes per store view
Before it always got it from the default store view